### PR TITLE
Add DQN doc

### LINKED ISSUE #1921: Docs page "Algorithms >(D)DQN -- {TensorFlow, PyTorch}"

### PR DESCRIPTION
**Pull request by maliesa96**
closes #1921

### CODE REVIEW

File: docs/user/algo_dqn.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+Both the Tensorflow and PyTorch branches of garage contain several DQN examples.
+
+### Pytorch
+
+The PyTorch branch provides an Atari launcher (see below) that allows you to easily run experiments from the command line like so:
+
+```bash
+# train on PongNoFrameskip-v4 for 10M steps
+python examples/torch/dqn_atari Pong --seed 22 --n_steps 10e6
```
Comment (ryanjulian) [inline docs/user/algo_dqn.md:62]: i don't think this line executes? also keep in mind that we should assume users are working from a pip distribution, not the development repo.

File: docs/user/algo_dqn.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                                 ...)
+```
+
+See the Atari launcher below for a full example.
+
+## Examples
+
+Both the Tensorflow and PyTorch branches of garage contain several DQN examples.
+
+### Pytorch
```
Comment (ryanjulian) [inline docs/user/algo_dqn.md:56]: can you include training curves for each of these examples?

File: docs/user/algo_dqn.md (hunk 1 above)
Comment (maliesa96) [reply docs/user/algo_dqn.md]: Fixed, this should run on the latest release.

File: docs/user/algo_dqn.md (hunk 2 above)
Comment (maliesa96) [reply docs/user/algo_dqn.md:56]: Sure, updated with the atari curves.

**Review by krzentner — APPROVED** [review]

**Review by ryanjulian — APPROVED** [review]
